Lib/ufoLib/glifLib.py

```diff
@@ -1243,8 +1242,7 @@ def _validateAndMassagePointStructures(contour, pointAttributes, openContourOffC
 		# we only care about how many offCurves there are before an onCurve
 		# filter out the trailing offCurves
 		offCurvesCount = len(contour) - 1 - lastOnCurvePoint
-		stripedContour = contour[:-offCurvesCount] if offCurvesCount else contour
-		for element in stripedContour:
+		for element in contour:
```

Comment:
are you no longer filtering out the trailing offcurves here?

Reply:
I don't think it's doing anything, the code for offcurves is:

if segmentType is None:
    offCurvesCount += 1

so it's just going to increment offCurvesCount, the code doesn't do anything until an oncurve appears.

Reply:
I see